app/templates/components/ui-table/cell/cell-simple-buttons.hbs

```diff
@@ -1,11 +1,25 @@
-<div class="ui vertical compact basic buttons">
-  {{#ui-popup content=(t 'View') class='ui icon button'  click=(action viewSession record.id) position='left center'}}
-    <i class="unhide icon"></i>
-  {{/ui-popup}}
-  {{#ui-popup content=(t 'Edit') class='ui icon button' click=(action editSession record.id record.event.id) position='left center'}}
-    <i class="edit icon"></i>
-  {{/ui-popup}}
-  {{#ui-popup content=(t 'Delete') click=(action (confirm (t 'Are you sure you would like to delete this Session?') (action deleteSession record))) class='ui icon button' position='left center'}}
-    <i class="trash outline icon"></i>
-  {{/ui-popup}}
-</div>
+{{#if device.isMobile}}
+  <div class="ui horizontal compact basic buttons">
+    {{#ui-popup content=(t 'View') class='ui icon button'  click=(action viewSession record.id) position='left center'}}
```

Comment:
Don't repeat the entire code in if and else.
There's only one word (horizontal/vertical) that is getting changed. Shouldn't you be comparing just for that?

Reply:
Please review @ritikamotwani